content/800-guides/240-partner-api.mdx

```diff
@@ -0,0 +1,193 @@
+---
+title: 'How to use the Prisma Postgres Partner API'
```

Comment:
I'd suggest a title along the lines of e.g. "Partner Database Provisioning & User Claim Flow"

Plus, more importantly, we should call the API the "Prisma Postgres Management API", so that it's clear that partners and endusers use the same API, not a different one. This should be updated throughout the guide.